packages/ptokens-assets-evm/src/ptokens-evm-provider.ts

```diff
@@ -188,12 +177,15 @@ export class pTokensEvmProvider implements pTokensAssetProvider {
       address: stringUtils.addHexPrefix(contractAddress),
       abi: abi,
       functionName: method,
-      args: _args
+      args: _args,
     }) as Promise<R>
   }
 
+  /**
+   * @deprecated Use https://viem.sh/docs/actions/public/waitForTransactionReceipt.html instead
+   */
   async waitForTransactionConfirmation(_tx: string, _pollingTime = 1000) {
```

Comment:
Why not remove the polling and use waitForTransactionReceipt directly inside this function?